crates/bevy_ui/src/render/mod.rs

```diff
@@ -222,30 +225,46 @@ const UI_CAMERA_FAR: f32 = 1000.0;
 // TODO: Evaluate if we still need this.
 const UI_CAMERA_TRANSFORM_OFFSET: f32 = -0.1;
 
-#[derive(Component)]
-pub struct DefaultCameraView(pub Entity);
+/// The UI camera used by this Camera's viewport.
+///
+/// This component is inserted into the render world in the
+/// [`extract_default_ui_camera_view`] system.
+///
+/// The component is attached to the "actual" viewport's camera.
+/// The UI camera's `ExtractedView` is attached to the entity in the
+/// `entity` field.
+#[derive(Component, Debug)]
+pub struct UiCamera {
+    /// The entity for the UI camera.
```

Comment:
How do we set this correctly?

Reply:
You shouldn't really. `UiCamera` only exists in the render world, I don't think this is intended for end-users. I kept it public since the original was.

This would be used by someone who _really_ wants to create their own UI rendering but re-use some of the bevy_ui rendering building blocks.

Reply:
Fair enough. Can you add docs to the struct explaining that "UiCamera only exists in the render world, I don't think this is intended for end-users"?